content/metadata.yaml

```diff
@@ -53,3 +53,16 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-23
+    github: nilswellhausen
```

Comment:
My comment is that I think you're missing the leading "-" before your info is added that denotes a new author. So it should look like:
```
  -
    github: nilswellhausen
```
If you can make that change, I will approve!

Reply:
Thank you! Casey and I fixed it now